pkg/schema/v1/config_map.go

```diff
@@ -73,6 +77,15 @@ func (c *ConfigMap) Obtain(k8s kmetav1.Object) {
 			LabelId:     labelId,
 		})
 	}
+
+	scheme := kruntime.NewScheme()
+	err := kcorev1.AddToScheme(scheme)
```

Comment:
Don't `panic` and explicitly ignore error, i.e. `_ = *.AddToScheme(scheme)`. Please adjust wherever applicable.